test/built-ins/Array/fromAsync/asyncitems-iterator-async.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.fromasync
+description: >
+  Array.fromAsync handles an async iterator returned from @@iterator
+includes: [asyncHelpers.js, compareArray.js, temporalHelpers.js]
+flags: [async]
+features: [Array.fromAsync]
+---*/
+
+asyncTest(async function () {
+  async function * asyncGen() {
+    for (let i = 0; i < 4; i++) {
+      yield Promise.resolve(i * 2);
+    }
+  }
+
+
+  return; // check what this should do
```

Comment:
Leftover from debugging?

Reply:
Turns out this scenario doesn't do anything interesting (besides timing out)

Reply:
So, should we just not have this file?

Reply:
Huh, I was sure I deleted it